app/views/spree/api/v1/orders/complete.v1.rabl

```diff
@@ -0,0 +1,7 @@
+child :available_payment_methods => :payment_methods do
```

Comment:
Can we abstract this to a higher level and introduce a partial that's used in state `payment` and `complete`?

Reply:
Since we are specifically templating for a specific state in the order show API response i would like this to be separated. 
The mollie gateway does not really lend itself for cross implementations and by including this hard overwrite of the complete partial (which is empty by default) we allow for easy readability of what is included per state and easy extensibility of functionality per order state. By combining the 2 into one abstract partial we lose this readability and extensibility and furthermore we are destroying the default spree functionality of including json data in state based partials.

Reply:
OK, agreed. Let's keep it like this 👍